### PR TITLE
Update ServiceLBNodePortControl feature gate status

### PR DESCRIPTION
**Pull request by hanlins**
Signed-off-by: Hanlin Shi <shihanlin9@gmail.com>

### CODE REVIEW

**Comment by hanlins** [comment]
For your reference, the k/k PR (https://github.com/kubernetes/kubernetes/pull/100412) is merged.

**Comment by tengqm** [comment]
- Please limit your change to English only;
- Please learn from existing features how the version history is preserved.

**Review by sftim — COMMENTED** [review]
Echoing https://github.com/kubernetes/website/pull/28770#issuecomment-873406882

- we do need this PR to only change English docs
   - the docs release team aren't able to resolve merge conflicts in other localizations, so we make sure not to provoke any such conflict
- you must also preserve the feature history (alpha phase) in the English docs

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -160,7 +160,8 @@ different Kubernetes components.
 | `RunAsGroup` | `true` | Beta | 1.14 | |
 | `SeccompDefault` | `false` | Alpha | 1.22 | |
 | `ServiceInternalTrafficPolicy` | `false` | Alpha | 1.21 | |
-| `ServiceLBNodePortControl` | `false` | Alpha | 1.20 | |
+| `ServiceLBNodePortControl` | `false` | Alpha | 1.21 | 1.21 |
```
Comment (tengqm) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:163]:
```suggestion
| `ServiceLBNodePortControl` | `false` | Alpha | 1.20 | 1.21 |
```

**Comment by PI-Victor** [comment]
the enhancement is marked as 'At Risk', holding till it's back to green.

/hold

**Comment by PI-Victor** [comment]
this enhancement is now officially back to green, we're good.

/hold cancel
/approve

**Pull request merged**